src/app/components/LiveHeaderMedia/index.test.tsx

```diff
@@ -159,4 +161,65 @@ describe('liveMediaStream', () => {
 
     expect(window.mediaPlayers.p0gh4n67.play).toHaveBeenCalledTimes(playCalls);
   });
+
+  describe('Event Tracking', () => {
+    beforeEach(() => {
+      jest.clearAllMocks();
+    });
+    const eventTrackingData = { componentName: 'live-header-media' };
+
+    describe('Click tracking', () => {
+      const clickTrackerSpy = jest.spyOn(clickTracking, 'default');
+
+      it('should not be enabled if event tracking data not provided', () => {
+        render(
+          <LiveHeaderMedia
+            mediaCollection={fixtureData as MediaCollection[]}
+          />,
+        );
+        const playCloseButton = screen.getByTestId('watch-now-close-button');
+        fireEvent.click(playCloseButton);
+        expect(clickTrackerSpy).toHaveBeenCalledWith(undefined);
+      });
+      it('should register click tracker if event tracking data provided', () => {
+        render(
+          <LiveHeaderMedia
+            mediaCollection={fixtureData as MediaCollection[]}
+            eventTrackingData={{
+              componentName: 'live-header-media',
+            }}
+          />,
+        );
+        expect(clickTrackerSpy).toHaveBeenCalledWith(eventTrackingData);
+        expect(clickTrackerSpy).toHaveBeenCalledTimes(1);
+      });
+    });
+
+    describe('View tracking', () => {
+      const viewTrackerSpy = jest.spyOn(viewTracking, 'default');
+
+      it('should not be enabled if event tracking data not provided', () => {
+        render(
+          <LiveHeaderMedia
+            mediaCollection={fixtureData as MediaCollection[]}
+          />,
+        );
+
+        expect(viewTrackerSpy).toHaveBeenCalledWith(undefined);
```

Comment:
As above - I would not have expected the view tracker to have been called if no data was passed...